pkg/agent/secrets.go

```diff
@@ -21,15 +19,15 @@ const (
 	secretSyncActionDelete = secretSyncAction("DELETE")
 )
 
-type secretInterface interface {
+type SecretInterface interface {
```

Comment:
Why is this being exported now? 

If it needs to be exported then it should have doc comments

Reply:
Because otherwise `func NewSecretsGetter(namespace string) (secretInterface, error)` returns a private interface, and it triggers a check from the IDE saying `Exported function with the unexported return type`